obsproject/obs-studio
# UI: Suppress multiple message boxes

### Description



While a message box is shown and the user has not closed it, another message box won't be shown unless it has a higher severity.

### Motivation and Context




When multiple errors happen and each error triggers a message box, it will create a huge number of popup dialogs.
On XFCE4 on Linux, this will results whole desktop not responding.

There is an issue #8603 reporting OBS crashes on Windows.

### How Has This Been Tested?




OS: Fedora 36

1. Set fmp4 format, enable replay buffer.
2. Start replay buffer
3. Wait for a while
4. Save the replay buffer
5. Try 3-4 several times until a dialog box appears.

### Types of changes



- Tweak (non-breaking change to improve existing functionality)





### Checklist:
